sor-api/src/main/java/com/bazaarvoice/emodb/sor/api/Compaction.java

```diff
@@ -38,32 +38,43 @@
     private Set<String> _lastTags;
 
     /**
-     * UUID of the most recent consistent delta which changed the resolved object deleted by compaction.
-     * Any delta between this and the cutoff are redundant.
+     * UUID of the most recent consistent delta which changed the resolved object deleted by compaction.  Any delta
+     * between this and the cutoff resolve to the same content, excluding intrinsics such as version and signature.
+     */
+     private final UUID _lastContentMutation;
+
+    /**
+     * Like {@link #_lastContentMutation} except this is the UUID of the most consistent delta which changed the
+     * resolved object or its metadata (namely, the event tags).  Any delta between this and the cutoff do not need to
```

Comment:
I feel an example would make it much clearer. 
For instance, a `no-op` delta would never update `_lastContentMutation`, but a `no-op` delta with a **new tag** would update `_lastMutation`, but not `_lastContentMutation`.